### PR TITLE
Prepare 1.26.0

### PR DESCRIPTION
**Pull request by jack-berg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,86 @@
...
+the [Logs Bridge API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/bridge-api.md)
+is _not_ meant for end users. Log appenders use the API to bridge logs from existing log
+frameworks (e.g. JUL, Log4j, SLf4J, Logback) into OpenTelemetry. Users configure the Log SDK to
+dictate how logs are processed and exported.
+
+`opentelemetry-opentracing-shim` is now stable!
+
+### SDK
+
+* Create OtelVersion class at build time. Use to resolve artifact version in `Resrouce`.
```
Comment (mateuszrzeszutek) [inline CHANGELOG.md:21]:
```suggestion
* Create OtelVersion class at build time. Use to resolve artifact version in `Resource`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,86 @@
...
+* Remove log record timestamp default.
+  ([#5374](https://github.com/open-telemetry/opentelemetry-java/pull/5374))
+* Align BatchLogRecordProcessor defaults with specification.
+  ([#5364](https://github.com/open-telemetry/opentelemetry-java/pull/5364))
+* Rename setEpoch to setTimestamp.
+  ([#5368](https://github.com/open-telemetry/opentelemetry-java/pull/5368))
+
+### Exporters
+
+* Add skaffolding for low level exporter TLS API.
```
Comment (mateuszrzeszutek) [inline CHANGELOG.md:42]: Typo?
```suggestion
* Add scaffolding for low level exporter TLS API.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,86 @@
 
 ## Unreleased
 
+This release represents the release candidate ("RC") release for the Logs Bridge API / SDK. In the
+following release, `opentelemetry-api-logs` will be merged into `opentelemetry-api`,
```
Comment (trask) [inline CHANGELOG.md:6]: (I initially read "following release" as the release that is described in the following text)

```suggestion
next release, `opentelemetry-api-logs` will be merged into `opentelemetry-api`,
```

**Review by mateuszrzeszutek — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,86 @@
...
+`opentelemetry-sdk-logs` will be marked as stable, `opentelemetry-exporter-otlp-logs` will be
+merged into `opentelemetry-exporter-otlp`, `opentelemetry-sdk-logs-testing` will be merged
+into `opentelemetry-sdk-testing`, `opentelemetry-sdk-extension-autoconfigure` will enable `otlp`
+exporter by default (i.e. `otel.logs.exporter=otlp`). For more details, see tracking
```
Comment (trask) [inline CHANGELOG.md:10]:
```suggestion
log exporter by default (i.e. `otel.logs.exporter=otlp`). For more details, see tracking
```

File: CHANGELOG.md (hunk 1 above)
Comment (trask) [inline CHANGELOG.md:21]:
```suggestion
* Create OtelVersion class at build time which is used to resolve artifact version in `Resource`.
```

**Review by jkwatson — APPROVED** [review]

**Pull request merged**